xbmc/cores/AudioEngine/Engines/ActiveAE/AudioDSPAddons/ActiveAEDSP.cpp

```diff
@@ -628,6 +648,19 @@ void CActiveAEDSP::UpdateAddons()
         }
       }
     }
+    else if (!bEnabled && IsKnownAudioDSPAddon(addon))
+    {
+      CLog::Log(LOGDEBUG, "Disabling AudioDSP add-on: %s", addon->ID().c_str());
+      AE_DSP_ADDON dspAddon = std::dynamic_pointer_cast<CActiveAEDSPAddon>(addon);
```

Comment:
@Paxxi 
Thx for the hint. I will check this for AudioDSP V2.0. As AudioDSP will be completely disabled I will avoid this check.